front/src/components/CTF/Guests.vue

```diff
@@ -1,39 +1,42 @@
 <template>
   <div class="q-gutter-md">
-    <div class="row">
-      <div class="col q-px-sm">
-        Invite guests to join <b>{{ ctf.title }}</b>
-      </div>
+    <div class="row text-h6">Invite guests</div>
+    <div class="row q-mt-sm">
+      Choose which guests can view and modify tasks in this CTF.
```

Comment:
I think this info is not complete. A guest can also access credentials so we should mention that here. And guests can access Discord channels if their account is linked and channels are created in a Discord server.